test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore3x/Microsoft.Test.E2E.AspNetCore3x.OData.csproj

```diff
@@ -31,6 +31,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="2.2.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.0" />
     <PackageReference Include="Microsoft.OData.Client" Version="7.6.1" />
+    <PackageReference Include="System.Diagnostics.EventLog" Version="4.7.0" />
```

Comment:
why do we need this package dependency?

Reply:
I've actually hasn't changed anything in code (.cs files), but existing

'.\WebApi\test\E2ETest\Microsoft.Test.E2E.AspNet.OData\QueryComposition\ServerTypeTests.cs'

is using System.Diagnostics, specifically EventLog and EventLogEntryType and I could not build the project (dotnet 3.1.2) freshly downloaded/checked out into a new empty folder (see build errors in my pull request and/or commit descriptions).
---> Note that
TargetFrameworkVersion=v4.6.1 for Microsoft.Test.E2E.AspNetCore.OData.csproj but
TargetFrameworkVersion=netcoreapp3.1 for Microsoft.Test.E2E.AspNetCore3x.OData.csproj.

The whole class Microsoft.Test.E2E.AspNet.OData.QueryComposition.ServerTypeTests
seems to be unused, test code compiles without it and test results with or without it are the same on my computer.
If ServerTypeTests.cs was removed, the PackageReference could be removed as well.
Still I am not confident/familiar enough with the project to propose the removal.

P.S.
The ServerTypeTests.cs was added here
Commit 20d0ab57
Fan Ouyang <t-oufan@microsoft.com>
28.04.2015 8:06:44
Committed by: 
Fan Ouyang <t-oufan@microsoft.com>
29.04.2015 8:51:34
Parent: 4bc08001